### DISCUSSION TITLE
eBesucher 0 points

### DISCUSSION THREAD

**Discussion by JonasKenke** [engageub/InternetIncome#183] [General]
Hey,
First of all thank you for that awesome script works really well.
I'm experiencing that I get only 0 point sites on eBesucher and read in a couple of forums that it's normal for Linux surfbars that they don't get any traffic.
Does it work for you?

**Comment by engageub** [comment]
Hello,
Thank you for the feedback.
If you get 0 points after surfing, it usually indicates that your IP is not detected as Residential IP.
You can also access the container using the mentioned URLs to see what is happening in the browser. 

Thank you
